dev-embedded/avr_libc/avr_libc-2.1.0.recipe

```diff
@@ -1,52 +1,49 @@
 SUMMARY="C library for Atmel AVR microcontrollers"
-DESCRIPTION="
-AVR Libc is a Free Software project whose goal is to provide a high quality C
+DESCRIPTION="AVR Libc is a Free Software project whose goal is to provide a high quality C \
 library for use with GCC on Atmel AVR microcontrollers.
 
-Together, avr-binutils, avr-gcc, and avr-libc form the heart of the Free
-Software toolchain for the Atmel AVR microcontrollers.
-"
+Together, avr-binutils, avr-gcc, and avr-libc form the heart of the Free Software toolchain for \
+the Atmel AVR microcontrollers."
 HOMEPAGE="https://www.nongnu.org/avr-libc/"
-LICENSE="BSD (3-clause)"
 COPYRIGHT="1999-2022 Keith Gudger, Bjoern Haase, Steinar Haugen, Peter Jansen,
 	Reinhard Jessich, Magnus Johansson, Artur Lipowski, Marek Michalkiewicz,
 	Colin O'Flynn, Bob Paddock, Reiner Patommel, Michael Rickman,
 	Theodore A. Roth, Juergen Schilling, Philip Soeberg, Anatoly Sokolov,
 	Nils Kristian Strom, Michael Stumpf, Stefan Swanepoel, Eric B. Weddington,
 	Joerg Wunsch, Dmitry Xmelkov, The Regents of the University of California."
+LICENSE="BSD (3-clause)"
+REVISION="2"
 SOURCE_URI="http://download.savannah.gnu.org/releases/avr-libc/avr-libc-$portVersion.tar.bz2"
 CHECKSUM_SHA256="0b84cee5c08b5d5cba67c36125e5aaa85251bc9accfba5773bfa87bc34b654e8"
-REVISION="1"
+SOURCE_DIR="avr-libc-$portVersion"
+
 ARCHITECTURES="any"
-SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	avr_libc$secondaryArchSuffix = $portVersion
+	avr_libc = $portVersion
 	"
 
 REQUIRES="
-	haiku$secondaryArchSuffix
+	haiku
 	"
 BUILD_REQUIRES="
 	"
 BUILD_PREREQUIRES="
-	haiku${secondaryArchSuffix}_devel	
+	haiku_devel
 	cmd:autoconf
 	cmd:automake
 	cmd:autoreconf
 	cmd:avr_as >= 2.40
 	cmd:avr_gcc >= 13.1
-	cmd:libtoolize$secondaryArchSuffix
```

Comment:
Does it still build on 32 bit? Otherwise this needs something like this:

```shell
    if [ "$targetArchitecture" == "x86_gcc2" ]; then
        BUILD_PREREQUIRES+="cmd:libtoolize_x86"
    else
        BUILD_PREREQUIRES+="cmd:libtoolize"
    fi
```

For BUILD_REQUIRES, BUILD_PREREQUIRES and TEST_REQUIRES, this can be done if needed because it doesn't affect the package.